src/Components/v2/AuctionTimer.tsx

```diff
@@ -41,7 +40,21 @@ export class AuctionTimer extends React.Component<Props> {
   }
 
   labelWithTimeRemaining() {
-    const display = moment(this.endDate).format("MMM D, h:mma")
```

Comment:
If there are many of these more complex formatting tasks it might be good to add a TODO to pull these out into a common datetime formatting file. 

I'm okay with that not happening in this PR though.

Reply:
Many of these formats are already in metaphysics which would be the most ideal. However removing moment and also updating all of the relay queries to pull in the correct formatting seamed like two separate tasks :)